### ISSUE TITLE
Receiving System-Emails "Connection to the other side was lost in a non-clean fashion: Connection lost."

### ISSUE THREAD

**Issue by peterknapp** [globaleaks/GlobaLeaks#3994]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

Other

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

Since yesterday we receive a high number of system-emails with this content:

> Platform: Hints
> Host: hints.cloud.newyorker.de (duirrnhho22vtd3cempvqsbrogtzxhnbnxglhzn6dqahguzmorm5qyid.onion)
> Version: 4.11.5
> twisted.web._newclient.ResponseNeverReceived
>     A L{ResponseFailed} that knows no response bytes at all have been received.
>     
> Traceback (most recent call last):
>   File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run
>     yield self.operation()
> twisted.web._newclient.ResponseNeverReceived: [<twisted.python.failure.Failure twisted.internet.error.ConnectionLost: Connection to the other side was lost in a non-clean fashion: Connection lost.>]
> 

Is this a known issue?

Best regards

### Proposed solution

Since we do not change the code, can you check it?

**Comment by evilaliv3** [comment]
Maybe you have blocked outgoing connections on the firewall and our update
check is not working?

**Comment by peterknapp** [comment]
You are correct. Turns out our network team altered our security "somehow" over the weekend.

Can you tell us the target addresses/ip's for the updater?
This way we can allow the targets in our firewall.
